meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,169 @@
+
+##### Peer ID Inlining
+
+Summary: While diagnosing the above textile issue, we (@whyrusleeping) found that we _would have_ broken Textile in the next release due to a change in how we're calculating peer IDs.
+
+@stebalien - We've now completely reverted to 0.4.18 behavior with a flag to restore 0.4.15 behavior. We can now cut a release.
+
+#### GraphSync
+Summary: Sync Graphs Quickly (without lots of requests) - https://github.com/ipld/specs/pull/75
+
+@eingenito - in design - We need a definitive spec for the initial MVP implementation that meets the needs of Filecoin but doesn't hamper us in the future. 
+
+@hannahhoward - yes writing code for go-graphsync but will be blocked in a couple days. also need to put out a proposal for revised wire protocol which I do feel capable of
+
+#### Process Improvements
+Summary: We're trying to get better working as a team.
+
+## Cross-team updates
+
+#### Testbed
+Summary: [Testlab repo](https://github.com/libp2p/testlab/pull/1) is live! I kind of bungled the whole "empty master commit with large PR off it" so the best move is to really look at the "prototype" branch.
+
+Request: Currently making some decisions about how to run [test scenarios](https://github.com/libp2p/testlab/issues/2). Would LOVE input.
+
+
+#### MultiWriter IPNS/Append-Only DAGs
+Summary: @aschmahmann Got some positive feedback/review on the work so far. Did a fairly thorough review of how gossipsub and peer-base do message propagation and trying to develop a reusable interface that Append-Only DAGs can use as well. More eyes always welcome!
+
+@aschmahmann Due to IPFS de-scoping/focusing on package managers I have much less DDC work to do. Will be reaching out to some of you over the next week or two to see how I can be of use.
+
+#### [Your Initiative Here]
+Summary: [Your summary here.]
+
+_@example - I moved this initiative forward by doing ..., and my next steps are ..._
+
+#### DHT in JS IPFS
+
+@alanshaw - We merged the DHT feature, it's great because JS <=> Go interop and content/peer discovery, thanks to @vasco-santos, @daviddias and loads more people for helping bring this over the line. It'll be in the [next JS IPFS release - 0.35](https://github.com/ipfs/js-ipfs/issues/1826)
+
+#### Tests in JS IPFS HTTP Client
+
+@alanshaw - `js-ipfs-http-client` tests take 5 mintues now! https://github.com/ipfs/js-ipfs-http-client/pull/942 thanks to the "test" profile and @Kubuxu. I'd like to know exactly what this profile does? Can anyone point me in the right direction?
+
+- This is what it does: https://github.com/ipfs/go-ipfs-config/blob/master/profile.go#L68-L85 (@Kubuxu)
+
+## Questions
+
+* Jenkins/Travis? (@eingenito)
+Turns out the go-ipfs project is okay to use CircleCI exclusively
+* Graphsync (@eingenito)
+Graphsync team is busy this week but will try to arrange some time to further discuss design
+* Base32/CIDv1 (@eingenito)
+Most important next milestone is storing/fetching multihashes in the DHT in libp2p
+
```

Comment:
This notes are amazing! It is amazing to see progress so clearly outlined. Awesome work Go Core Team 👏🏽👏🏽👏🏽👏🏽